src/components/Article/Article.css

```
@@ -1 +1,30 @@
-/* Put article styles here */
+.article {
+  margin: 5em auto 0;
+  padding: 1em;
+  font-size: 140%;
+  max-width: 800px;
+  background: white;
+  box-shadow: rgba(0, 0, 0, 0.05) 0 3px 15px;
+}
+
+.__paragraph {
```

```suggestion
.article__paragraph {
```
Misspelled classname